### DISCUSSION TITLE
www.radio-browser.info natively via upmpdcli

### DISCUSSION THREAD

**Discussion by kan-ibal** [rern/rAudio#1293] [Feature Requests]
Upmpdcli has already a new media server plugin for accessing https://www.radio-browser.info/. I was able to make it run in rAudio by installing pyradios python module:
`pip install pyradios`
and adding "radio-browseruser = upmpdcli" to the /etc/upmpdcli.conf. 
`echo "radio-browseruser = upmpdcli" >> /etc/upmpdcli.conf`
Now I can access radio-browser stations and add it to playlist using UPnP client ( e.g. BubbleUPnP or VLC).
What I can think of is there any way to implement front end for that in rAudio?

**Comment by ceejayemm** [comment]
Plus 1 from me.

I have been playing around with this in VLC and it seems to answer the problems I have been having with getting access to the Bauer Group stations (Magic UK, Greatest Hits Radio et al) which have a 'special' url which incorporates a daily ID change.  If this could be incorporated into rAudio with a usable UI then this could be a very interesting proposition and possibly remove the need for special addons for Radio Paradise, France Musique etc.

Chris

**Comment by kan-ibal** [comment]
Via VLC you can only browse radio stations. Using BubbleUPnP you can add station to rAudio render and play it in rAudio.

**Comment by ceejayemm** [comment]
That's why I noted the need for a decent UI in rAudio to make the necessary selections, I would not want to have to use a separate application (BubbleUPnP etc) to make use of this facility. I only tried it out in VLC to see what it could do. If it can be made to work entirely within rAudio I think this could be a killer addition. BTW, as I live in the UK access to the BBC Sounds app output via rAudio would also be such a massive bonus.

Chris

**Comment by Raitsa** [comment]
Plus one from here to the suggestion.
Looks like OP has found maybe a good way to make this a reality for rAudio.  
radio-browser.info is amazing asset, hopefully we get to enjoy it.

**Comment by AndyFx12** [comment]
Alternative you can use RadioDroid (based on radio-browser.info) for Android. It can play radios via rAudio' s MPD. So played radios will be visible at playlist. Contextmenu of unknown stations gives you an option to add them to your library.
regards, Andy
